hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1455,6 +1460,13 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	// Reconcile the ETCD Recovery member
+	if hcluster.Spec.Etcd.ManagementType == hyperv1.Managed {
+		if err := r.reconcileETCDMemberRecovery(ctx, hcluster, createOrUpdate); err != nil {
+			return ctrl.Result{}, fmt.Errorf("failed to reconcile etcd recovery member: %w", err)
+		}
+	}
```

Comment:
The PR description still mentions an `hypershift.openshift.io/etcd-restore-failing-member` annotation, did this change in this iteration of this PR? I don't see the annotation anymore :)